zkevm-circuits/src/evm_circuit/util/common_gadget.rs

```diff
@@ -901,12 +889,6 @@ impl<F: Field> SstoreGasGadget<F> {
         original_value: eth_types::Word,
         is_warm: bool,
     ) -> Result<(), Error> {
-        self.value
-            .assign(region, offset, Some(value.to_le_bytes()))?;
-        self.value_prev
-            .assign(region, offset, Some(value_prev.to_le_bytes()))?;
-        self.original_value
-            .assign(region, offset, Some(original_value.to_le_bytes()))?;
```

Comment:
Perhaps these assignments were duplicated?

Reply:
yes they are duplicate assignment. And those 3 cell(s) are passed from caller via constructor. Assignment them should be the caller responsibility.